test/zdtm/static/netns-nft.checkskip

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-test -f /usr/sbin/nft || exit 1
+pkg-config --libs libnftables
```

Comment:
Would the following workaround become redundant after this change?
https://github.com/checkpoint-restore/criu/blob/e49ce513c8df43e8d3b30b7acfbe7bd496d89862/.cirrus.yml#L49

Reply:
@rst0git yes